plugins/src/ios/OptimoveSDKPlugin.swift

```diff
@@ -1,14 +1,51 @@
-@objc(initialize:)
-  func initialize(command: CDVInvokedUrlCommand) {
-    let args = command.arguments
-    
-    let inAppConsentStrategy: InAppConsentStrategy = {
-      if let strategy = args?[2] as? String {
-        return .init(rawValue: strategy)!
-      }
-      return.notEnabled
-    }()
-
-    let config = OptimoveConfigBuilder(optimoveCredentials: args?[0] as? String, optimoveCredentials: args?[1] as? String).enableInAppMessaging(inAppConsentStrategy: inAppConsentStrategy).build()
-    Optimove.initialize(with: config)
-  }
\ No newline at end of file
+import OptimoveSDK
+import NotificationCenter
+@objc(Optimobile_Cordova) class OptimobileSDKPlugin : CDVPlugin {
```

Comment:
Naming: can we use Optimove instead of Optimobile?

Reply:
done

Reply:
Think we maybe missed a ref to `OptimobileSDKPlugin`

Reply:
I didn't understand that comment

Reply:
Ah sorry I just meant the class name is still Optimobile, can be renamed too :)

Reply:
done